src/components/editMeasure/EditMeasureNav.tsx

```diff
@@ -37,6 +41,16 @@ const EditMeasureNav = ({ isQDM }) => {
     navigate(`/measures/${id}/edit/details/`);
   }
   const match = useMatch("/measures/:id/edit/*")?.params?.["*"].split("/")[0];
+
+  useEffect(() => {
+    if (id) {
+      testCaseService.current
+        .getTestCasesByMeasureId(id)
+        .then((testCaseList) => setTestCaseLength(testCaseList.length))
+        .catch((error) => {});
```

Comment:
Are we intentionally ignoring the exception ? We could ask Michael and see what should be displayed in place of count if the actual data couldn't be displayed

Reply:
Michael says that it might be confusing to the user, if there's an error, then there'll be an error getting the test cases anyway